<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="TraceViewWidget" name="main_trace_view">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>19</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_trace">
    <property name="title">
     <string>&amp;Trace</string>
    </property>
    <addaction name="action_capture"/>
    <addaction name="action_capture_filter"/>
    <addaction name="action_clear"/>
    <addaction name="action_clean_on_new_connect"/>
    <addaction name="separator"/>
    <addaction name="action_session_manager"/>
    <addaction name="separator"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_settings"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_trace"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_session_manager">
   <property name="text">
    <string>Session Manager...</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_capture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/start</normaloff>
     <normalon>:/icons/stop</normalon>:/icons/start</iconset>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="action_capture_filter">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/capture_filter</normaloff>:/icons/capture_filter</iconset>
   </property>
   <property name="text">
    <string>Capture Filter...</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save Trace...</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open Trace...</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/clear_trace</normaloff>:/icons/clear_trace</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About Trace-X...</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="icon">
    <iconset resource="app.qrc">
     <normaloff>:/icons/settings_white</normaloff>:/icons/settings_white</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="action_clean_on_new_connect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clean On New Connect</string>
   </property>
   <property name="toolTip">
    <string>Clean trace on new connection</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TraceViewWidget</class>
   <extends>QFrame</extends>
   <header>trace_view_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
